boa/benches/parser.rs

```diff
@@ -100,11 +85,7 @@ fn long_file_parser(c: &mut Criterion) {
         b.iter(|| {
             let file_str = fs::read_to_string(FILE_NAME)
                 .unwrap_or_else(|_| panic!("could not read {}", FILE_NAME));
-
-            let mut lexer = Lexer::new(black_box(&file_str));
-            lexer.lex().expect("failed to lex");
-
-            Parser::new(&black_box(lexer.tokens)).parse_all()
+            Parser::new(black_box(file_str.as_bytes())).parse_all()
         })
     });
```

Comment:
we don't have to `read_to_string` we can just pass the `File` because it implements `Read` trait:
```rust
   let file = std::fs::File::open(FILE_NAME).expect("Could not open file");
    c.bench_function("Long file (Parser)", move |b| {
		b.iter(|| {
            Parser::new(black_box(file)).parse_all()
        })
    });
```

Reply:
Done